client/core/locale_ntfn.go

```diff
@@ -60,14 +60,14 @@ var enUS = map[Topic]*translation{
 		template: "%v wallet has no network peers!",
 	},
 	// [ticker, error]
-	TopicWithdrawError: {
-		subject:  "Withdraw error",
-		template: "Error encountered during %s withdraw: %v",
+	TopicSendError: {
+		subject:  "Send error",
+		template: "Error encountered while sending %s: %v",
 	},
 	// [ticker, coin ID]
-	TopicWithdrawSend: {
-		subject:  "Withdraw sent",
-		template: "Withdraw of %s has completed successfully. Coin ID = %s",
+	TopicSendSuccess: {
+		subject:  "Send Successful",
+		template: "Sending %s has completed successfully. Coin ID = %s",
```

Comment:
For another day, but I don't quite understand why these are notifications at all.  User clicks and the HTTP response contains the success/error message, which can go on the form, not the notifications.   Strange...

I'm also not entirely sure we need to change these note type and topic variable names, or the string literals assigned to them.  I guess there's no special handling for `NoteTypeWithdraw = "withdraw"` (now `NoteTypeSend = "send"`) anywhere on the frontend, so I suppose it's fine.  I guess just check closely and keep the new names and strings, but it feels like unneeded churn since none of it is user-facing and it was all still clear an applicable despite the updated capability (optionally subtracting fees).

Perhaps @buck54321 has some thoughts about both of the above points.

Reply:
These are sent at level `Success`. I think the idea is that it's nice to have a notification for certain important actions.

Reply:
Yeah, that's true about the success ntfn.  I like to have the txid in the notification menu to be copied at any time.

About the error one though, we've been moving to have those on the form itself, so isn't that a bit of a relic?

Reply:
> we've been moving to have those on the form itself, so isn't that a bit of a relic?

Are we removing the error notification for withdraw/send? True, the form displays the error if any. But it looks like it would be nice to have withdraw/send errors in the notifications (at least they don't just disappear, so users have a persistent record of errors related to withdrawing/sending their funds.)

Reply:
Leaving I guess.  Personally, I think it's bizarre for the error notification to show up in the tray _and_ on the form.  What's the point of a persistent record of a failed withdraw request if nothing was withdrawn.  The user will resolve the error and try again, and the success message can stick around so they can find the txid.